client/post-editor/media-modal/image-editor/index.jsx

```diff
@@ -49,30 +48,24 @@ const MediaModalImageEditor = React.createClass( {
 			fileName = 'default',
 			mimeType = 'image/png';
 
-		if ( this.props.items && this.props.items[ this.props.selectedIndex ] ) {
-			src = MediaUtils.url( this.props.items[ this.props.selectedIndex ], {
+		const media = this.props.items[ this.props.selectedIndex ];
+
+		if ( this.props.items && media ) {
```

Comment:
`this.props.items` condition redundant at this point if you're already accessing it in the line above?